redaxo/src/addons/structure/lib/api_functions/api_article_move.php

```diff
@@ -0,0 +1,28 @@
+<?php
+/**
+ * @package redaxo\structure
+ */
+class rex_api_article_move extends rex_api_function
```

Comment:
diesen code hast du aus einer bestehenden funktion extrahiert, oder sollte der durch das aufteilen des codes im anderen PR eigentlich abgehandelt sein?

Reply:
Nicht ganz. Die api_function article_move fehlte bisher noch komplett.
Die Funktionen wurden bisher im Content-Plugin behandelt. Dort werden die api_functions größtenteils ignoriert, weil die Funktionalität direkt in die Seite integriert ist: https://github.com/redaxo/redaxo/blob/master/redaxo/src/addons/structure/plugins/content/pages/content.php#L350-L413
Ich habe auch festgestellt, dass mindestens eine der api_functions noch buggy ist, weil sie wohl nie genutzt wurde. Das muss ich noch anpassen.

Reply:
können wir das ebenfalls in einem separaten PR machen?

Reply:
OK, mach ich.

Reply:
Ich habe alle Funktionen der Seite in api_functions umgewandelt, bzw. den Aufruf der vorhandenen api_functions eingebunden. Einige waren fehlerhaft, da sie anscheinend nie genutzt wurden. Ich habe daher die Funktion der Content -Seite übernommen, die funktioniert: #1425.